tei_adapter/tei_id_extractor/src/main/scala/weco/catalogue/tei/id_extractor/GitHubBlobReader.scala

```diff
@@ -0,0 +1,71 @@
+package weco.catalogue.tei.id_extractor
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers._
+import akka.http.scaladsl.unmarshalling.{Unmarshal, Unmarshaller}
+import uk.ac.wellcome.json.JsonUtil._
+
+import java.net.URI
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
+
+class GitHubBlobReader(token: String)(implicit ac: ActorSystem) {
+  implicit val ec = ac.dispatcher
+  def getBlob(uri: URI): Future[String] = {
+    val request = HttpRequest(
+      uri = Uri(uri.toString),
+      headers = List(
+        Accept(
+          MediaType.applicationWithFixedCharset(
+            "vnd.github.v3+json",
+            HttpCharsets.`UTF-8`)),
+        `Cache-Control`(CacheDirectives.`no-cache`),
+        Connection("keep-alive"),
+        Authorization(OAuth2BearerToken(token))
```

Comment:
Actually I don't think we do. The github documentation recommends - but doesn't require - the Accept header with the api version, so that things don't shift under your feet. EVerything else was me trying to understand why I was getting 403 and it turned out it was rate limiting.
I will delete the ones we don't need and comment the one we do need